CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -304,9 +331,16 @@ public static int StaticInitialise(CVKDevice cvkDevice) {
     
     public CVKAxesRenderable(CVKVisualProcessor visualProcessor) {
         parent = visualProcessor;
-      
     }
     
+    
+    public void DestroyStatic() {
+        vertexShaderSPIRV.free();
```

Comment:
What's calling this?  Freed resources should be nulled so Java can clean up anything it allocated for that resource.  We also need to reset the static initialised flag too.

Reply:
We don't have a good place to call this atm, ideally it would be when constellation starts and then shutsdown.  For now I'll call when the graph is destroyed and add a comment.